client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -66,6 +68,7 @@ export const MySitesSidebarUnifiedMenu = ( {
 	}, [ selected, childIsSelected, reduxDispatch, sectionId, sidebarCollapsed ] );
 
 	const onClick = () => {
+		trackMenuItemClick( identifier );
```

Comment:
See my comment in https://github.com/Automattic/jetpack/pull/19261#issuecomment-810063220. I think we should track the click only when a new page is opened. There are cases (width <= 782px) when this is will only expand/collapse the section which we don't need to track.